requirements.txt

```diff
@@ -9,15 +9,15 @@ opencv-python
 pandas
 pillow
 projectq
-qiskit
-qiskit-aer
+qiskit ; python_version < '3.12'
+qiskit-aer ; python_version < '3.12'
 requests
 rich
 scikit-fuzzy
 scikit-learn
 statsmodels
 sympy
-tensorflow
+tensorflow ; python_version < '3.12'
```

Comment:
I think we should open a tracking issue for remaining dependencies and mark it as "good first issue".